src/common/SurgeStorage.cpp

```diff
@@ -2752,3 +2752,35 @@ string findReplaceSubstring(string &source, const string &from, const string &to
 
 } // namespace Storage
 } // namespace Surge
+
+ScenesOutputData::ScenesOutputData()
```

Comment:
Why isn't this inside the Surge::Storage namespace?

Reply:
Moved inside SurgeStorage